src/common/helpers/request.ts

```diff
@@ -31,10 +31,9 @@ client.interceptors.response.use(
 
     if (
       error.response?.status &&
-      error.response.status !== 401 &&
       error.response.status !== 412 &&
       error.response.status !== 422 &&
-      error.response.status !== 400 && // Temporary solution. If you see this in few months, please delete (live previews).
```

Comment:
@beganovich @turbo124 I know that we had some issues with a 400 error on live_previews, so this was a temporary solution. Have we resolved it now, so I can remove the unnecessary checks?

Reply:
@beganovich it should be ok to remove line 37 now if we have toast, correct?

Reply:
I think so, yep 👍️

Reply:
@beganovich Okay, I just removed the checks for live previews.